src/viam/sdk/robot/client.cpp

```diff
@@ -354,7 +354,7 @@ std::shared_ptr<RobotClient> RobotClient::at_local_socket(const std::string& add
     const std::string addr = "unix://" + address;
     const char* uri = addr.c_str();
     const std::shared_ptr<grpc::Channel> channel =
-        grpc::CreateChannel(uri, grpc::InsecureChannelCredentials());
+        grpc::CreateCustomChannel(uri, grpc::InsecureChannelCredentials(), default_channel_args());
```

Comment:
I like this idea. Do we have any SDK private modules outside of `components/private` ? Maybe this could be in `rpc/private`? Or whatever else you had in mind

Reply:
Yeah, I think `rpc/private` is the right place.